### PR TITLE
[BUGFIX release] Backport autotracking bugfixes

### PR DESCRIPTION
**Pull request by pzuraq**

### CODE REVIEW

File: packages/@ember/-internals/utils/lib/mandatory-setter.ts [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import { assert } from '@ember/debug';
+import { _WeakSet as WeakSet } from '@ember/polyfills';
 import { DEBUG } from '@glimmer/env';
+import { Tag } from '@glimmer/validator';
```
Comment (rwjblue) [inline packages/@ember/-internals/utils/lib/mandatory-setter.ts]: I don't think `@glimmer/validator` exists in this branch (this fails the linting)

**Review by rwjblue — CHANGES_REQUESTED** [review]

File: packages/@ember/-internals/utils/lib/mandatory-setter.ts (hunk 1 above)
Comment (pzuraq) [reply packages/@ember/-internals/utils/lib/mandatory-setter.ts]: Ah, yeah, strange that tests are passing (I ran them locally)

Comment (rwjblue) [reply packages/@ember/-internals/utils/lib/mandatory-setter.ts]: It’s because it’s only used for types, not runtime

Comment (rwjblue) [reply packages/@ember/-internals/utils/lib/mandatory-setter.ts]: Rebased and fixed the import, verified linting passes...

**Review by rwjblue — APPROVED** [review]

**Pull request merged**